### ISSUE TITLE
cpptraj libreadline.so undefined reference

### ISSUE THREAD

**Issue by tluchko** [Amber-MD/cmake-buildscripts#120]
I have started getting these errors right at the end of my build

```
../../../../CMakeFiles/miniconda/install/lib/libreadline.so: undefined reference to `tgetstr'
../../../../CMakeFiles/miniconda/install/lib/libreadline.so: undefined reference to `tputs'
../../../../CMakeFiles/miniconda/install/lib/libreadline.so: undefined reference to `BC'
../../../../CMakeFiles/miniconda/install/lib/libreadline.so: undefined reference to `tgetent'
../../../../CMakeFiles/miniconda/install/lib/libreadline.so: undefined reference to `tgetflag'
../../../../CMakeFiles/miniconda/install/lib/libreadline.so: undefined reference to `tgoto'
../../../../CMakeFiles/miniconda/install/lib/libreadline.so: undefined reference to `UP'
../../../../CMakeFiles/miniconda/install/lib/libreadline.so: undefined reference to `tgetnum'
../../../../CMakeFiles/miniconda/install/lib/libreadline.so: undefined reference to `PC'
```

I know this looks like #56 but I just saw this today.  It happens on my cluster (CentOS 6.6), comet.sdsc.edu (CentOS 6.7) but not on my mac (10.13.4).

Attached are the log files from my cluster.
[errorreport.tar.gz](https://github.com/Amber-MD/cmake-buildscripts/files/2002884/errorreport.tar.gz)

**Comment by drroe** [comment]
I've come across this issues before. I solved it in the standalone configure by ensuring libtermcap is linked against when using an external readline (i.e. add `-ltermcap` after the readline library when linking); [see here](https://github.com/Amber-MD/cpptraj/blob/c66e08de66a8f92c07e42361064834b0afb52f8e/configure#L782-L786). I'm not sure how to do the equivalent in cmake, but I'm sure that @multiplemonomials knows how!

**Comment by multiplemonomials** [comment]
This should be a fairly straightforward fix, yeah.  However, I just moved and my house doesn't have internet yet, so it's going to be a few days.  In the meantime, a workaround should be `-DFORCE_INTERNAL_LIBS=readline`.

This issue probably started because, in the most recent version of this submodule, I enabled searching for libraries inside the internal Miniconda's library dirs, and Anaconda's readline does seem to have this problem
